astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -23,212 +168,638 @@ class NDArithmeticMixin(object):
     When subclassing, be sure to list the superclasses in the correct order
```

Comment:
The MRO dictates the order - a helpful reference is http://nedbatchelder.com/blog/201210/multiple_inheritance_is_hard.html and the comments underneath.

If the mixin overrides a method in `NDDataBase` you want the mixin listed before the base class.

It might be better to simply point out that order matters and provide a reference....